telegram/ext/_defaults.py

```diff
@@ -178,11 +209,11 @@ def disable_notification(self, value: object) -> NoReturn:
         )
 
     @property
-    def disable_web_page_preview(self) -> Optional[bool]:
-        """:obj:`bool`: Optional. Disables link previews for links in this
-        message.
+    def disable_web_page_preview(self) -> ODVInput[bool]:
+        """:obj:`bool`: Optional. Disables link previews for links in all outgoing
+        messages.
```

Comment:
your point of needing to build `Defaults` only once makes me want to deprecate DWPP. I'll add the depr note to the attr docstring but I don't think the attribute access warning is needed since we already warn when they build the Defaults object (in the `__init__`)